frame/benchmarking/src/tests.rs

```diff
@@ -105,163 +111,177 @@ parameter_types!{
 	pub const UpperBound: u32 = 100;
 }
 
-impl Config for Test {
-	type Event = ();
+impl pallet_test::Config for Test {
+	type Event = Event;
 	type LowerBound = LowerBound;
 	type UpperBound = UpperBound;
 }
 
-impl OtherConfig for Test {
-	type OtherEvent = ();
+impl pallet_test::OtherConfig for Test {
+	type OtherEvent = Event;
 }
 
 fn new_test_ext() -> sp_io::TestExternalities {
-	frame_system::GenesisConfig::default().build_storage::<Test>().unwrap().into()
+	GenesisConfig::default().build_storage().unwrap().into()
 }
 
-benchmarks!{
-	where_clause { where <T as OtherConfig>::OtherEvent: Into<<T as Config>::Event> }
-
-	set_value {
-		let b in 1 .. 1000;
-		let caller = account::<T::AccountId>("caller", 0, 0);
-	}: _ (RawOrigin::Signed(caller), b.into())
-	verify {
-		assert_eq!(Value::get(), Some(b));
-	}
+mod benchmarks {
```

Comment:
here benchmark macro is called in its own module because benchmark macro uses in scope `Call` for the pallet call. But construct_runtime already defines the outer `Call`.

the benchmark call and the test below haven't been changed